src/integrations/sentry_integration_qt.h

```diff
@@ -0,0 +1,15 @@
+#ifndef SENTRY_INTEGRATION_QT_H_INCLUDED
+#define SENTRY_INTEGRATION_QT_H_INCLUDED
+
+#ifdef __cplusplus
```

Comment:
I think its fine this way, but would rather wrap the function in the `.cpp` file in an `extern "C"` block, similar to how the other parts that depend on C++ work.